packages/components/pages/f-contact-preferences/src/components/ContactPreferences.vue

```diff
@@ -257,6 +271,15 @@ export default {
 </script>
 
 <style lang="scss" module>
+
+.c-contactPreferences-form-field {
+    margin-top: 0 !important;
```

Comment:
Would be good to understand why you need the `!important` here and below – does the styling come down from the component otherwise and that takes precedence? If so, which rule is causing the issue, as usually this indicates a problem in the way the component defines its styling.

Reply:
`!important` is used by `f-form-field` in several places:

![image](https://user-images.githubusercontent.com/20644043/144606851-7245670f-7f3c-4a21-8fc7-819832f313e6.png)

Reply:
Would be good if we could find a way to avoid using `!important` inside the form field component.

Reply:
Agree can this be treated as a separate issue?

Reply:
So the reason the important is on these, is that utility/trump classes like that shouldn't be overwritten (otherwise, there's no point in it being a trump class).

I'd suggest removing these trump classes from f-form-field label, as if they aren't always the value we want to use, they shouldn't be applied and will cause issues.

Reply:
https://github.com/justeat/fozzie-components/pull/1531

Reply:
`f-form-field` has been updated and the use of `!important` removed from this pr :)